transformers4rec/torch/model/base.py

```diff
@@ -619,7 +611,7 @@ def forward(self, inputs, *args, **kwargs):
                 return self.parent(inputs, *args, **kwargs)
 
             def training_step(self, batch, batch_idx):
-                loss = self.parent.compute_loss(*batch)
+                loss = self.parent(*batch, training=True, testing=False)
```

Comment:
do we need to specify the key `loss` here? 
```suggestion
                loss = self.parent(*batch, training=True, testing=False)['loss']
```

Reply:
Yes, changed it.